plugins/tech-radar/src/api.ts

```diff
@@ -34,7 +34,7 @@ export interface RadarQuadrant {
 export interface RadarEntry {
   key: string; // react key
   id: string;
-  moved: number;
+  moved: -1 | 0 | 1;
```

Comment:
Nice! We could potentially make this a TypeScript enum for even more readability?

```ts
enum MovedState {
  Down = -1,
  NoChange = 0,
  Up = 1
}
```

Then this could be something like

```suggestion
  moved: MovedState;
```